db/schema.rb

```diff
@@ -10,59 +10,8 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2021_09_15_095847) do
-
-  create_sequence "application_choices_id_seq"
```

Comment:
🔥

Reply:
Not sure if you've discussed this.

This has all gone because of the `ar-sequence` update. It will now only dump sequences that were manually created by the gem, but it's not backwards compatible. So any manually generated sequences with an old version of `ar-sequence` are not dumped.

Can we reinstate just the `qualifications_public_id_seq` (manually recreate it locally and then do a schema dump)

Reply:
Thanks for pointing this out, hadn't realised the change is not backwards compatible. Have restored this specific sequence in `db/schema.rb` with a new migration.